pkg/errors/v1alpha1/errors.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/pkg/errors"
+)
+
+// New is a wrapper over errprs.New()
+// It creates a new error for the given message.
+func New(message string) error {
+	if message == "" {
+		return nil
+	}
+	return errors.New(message)
+}
+
+// Errorf is a wrapper over errprs.New()
+// It creates a new error for the given
+// message and format.
+func Errorf(format string, args ...interface{}) error {
+	if format == "" {
+		return nil
+	}
+	return errors.Errorf(format, args...)
+}
+
+// Errors returns an error for a list of given errors.
+// if list length is empty then it returns nil.
+func Errors(errs []error) error {
+	if len(errs) == 0 {
+		return nil
+	}
+	var e Errs = errs
+	return &e
```

Comment:
return type is error, but returning &Errs.. is that fine?

Reply:
It is ok. here error in interface type